keps/sig-node/127-user-namespaces/README.md

```diff
@@ -270,15 +275,20 @@ we don't want to forget or want to highlight it. Some things that are obvious we
 need to tackle are not listed. Let us know if you think it is important to add
 something else to this list:
 
-- We will start with mappings of 64K. Tim Hockin, however, has expressed
+- We will start with mappings of 64K. Tim Hockin, however, have expressed
```

Comment:
should this still be `has`?

Reply:
ouch, yes. Will fix that and update the upstream PR. Thanks!